src/main/java/mate/academy/service/MovieServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package mate.academy.service;
+
+import mate.academy.dao.MovieDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.Movie;
+
+@Service
+public class MovieServiceImpl implements MovieService {
+    @Inject
+    private MovieDao movieDao;
+
+    @Override
+    public Movie add(Movie movie) {
+        return movieDao.add(movie);
+    }
+
+    @Override
+    public Movie get(Long id) {
+        return movieDao.get(id)
+                .orElseThrow(() -> new RuntimeException("Can`t get movie by id :" + id));
```

Comment:
```suggestion
                .orElseThrow(() -> new RuntimeException("Can`t get movie by id :" + id));
```

NoSuchElementException is better